codemods/qs/picocolors/index.js

```diff
@@ -0,0 +1,296 @@
+import { ts } from '@ast-grep/napi';
```

Comment:
this file should be in `codemods/qs/picoquery` rather than `codemods/qs/picocolors`